lib/prop/leaky_bucket_strategy.rb

```diff
@@ -7,21 +7,27 @@ module Prop
   class LeakyBucketStrategy
     class << self
       def counter(cache_key, options)
-        update_bucket(cache_key, options[:interval], options[:threshold]).merge(burst_rate: options[:burst_rate])
```

Comment:
previously we persisted the burst_rate ... which is pointless since it's passed on each call